src/main/resources/page/js/home-page-loader.js

```diff
@@ -0,0 +1,21 @@
+class HomePageLoader {
+    constructor() { }
+
+    load() {
+        this.loadHeader();
+        this.loadFooter();
+    }
+
+    loadHeader() {
+        NavBarTogglerButton = new NavBarTogglerButton();
+        NavBarTogglerButton.make();
```

Comment:
This is confusing, the variable name should start with a lowercase character
Besides it, could you please use the same pattern for these classes?
If every class would have a "make" method, and the class names would have the "maker" postfix, ButtonMaker and there is a make method, this would be really nice!

Reply:
I will rename it!
Good idea I will do it!